### PR TITLE
o-comments v6.0.0

### PR DESCRIPTION
**Pull request by ker-an**
This PR replaces `o-comments` v5 with `o-comments` v6 as the active version.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# The Origami Core team is the default owner of all Origami-supported repositories.
-# This file adds the owner as a reviewer for all pull requests made to this repository.
-# It references owners by source order - the last matching pattern takes the most precedence.
```
Comment (notlee) [inline .github/CODEOWNERS]: What was the reason for removing `.github/CODEOWNERS` and `.github/ISSUE_TEMPLATE.md`? Maybe good to reinstate and set to a customer products team?

File: src/scss/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/// Output All oComments Features
+@mixin oComments ($class: 'o-comments') {
```
Comment (notlee) [inline src/scss/_mixins.scss]:
```suggestion
@mixin oComments () {
```

The class argument is never used and we technically can't remove it later without it being a breaking change

File: README.md [hunk 3]
```diff
@@ -1,513 +1,210 @@
...
 
-```scss
-$o-comments-include-fonts: false;
-```
+Encoded URL: `https://www.ft.com/__origami/service/build/v2/bundles/css?modules%3Do-comments%406.0.0-beta.24%3A%2Fsrc%2Fscss%2Fcoral-talk-iframe%2Fmain.scss`
 
-## <div id="browser"></div> Browser support
-Works in accordance with our [support policy](https://docs.google.com/document/d/1mByh6sT8zI4XRyPKqWVsC2jUfXHZvhshS5SlHErWjXU).
+## Troubleshooting
+_This is a good place to put problems that come up repeatedly_
```
Comment (notlee) [inline README.md]:
```suggestion
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -22,7 +22,15 @@ jobs:
       - checkout
       - run: npx occ ${CIRCLE_TAG##v}
       - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > $HOME/.npmrc
```
[the remark this answers is not in the document]
Comment (ker-an) [reply .circleci/config.yml]: No problem! It looks like the `master` branch is using origami-ci so I will copy/paste it to `v6`.

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to a26890bbfeaa89375e28010cc080020731503710
js: 70.41kb decrease (-17.90kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to f51f1de5f393d8a7cef69cf1a118bbf3f4f157e0
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to 368f244b089262eee979536449a905ca02f5126d
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to 4259fa5b70b734ce68da5fcf83c105594d1e53ea
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to 4259fa5b70b734ce68da5fcf83c105594d1e53ea
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

File: .github/CODEOWNERS (hunk 1 above)
Comment (notlee) [reply .github/CODEOWNERS]: It's not! Thanks I'll create an issue to add it

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to c94ef8a5afc28558d5b3a684819b9b1c2685f8b5
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to e24ab9e592e951ee90273d6f7148c4703592ea74
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to e24ab9e592e951ee90273d6f7148c4703592ea74
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

File: origami.json [hunk 5]
```diff
@@ -1,61 +1,48 @@
...
 	"origamiVersion": 1,
-	"support": "https://jira.ft.com/secure/RapidBoard.jspa?rapidView=57&view=detail",
+	"support": "https://github.com/Financial-Times/o-comments/issues",
 	"supportContact": {
 		"email": "origami.support@ft.com",
 		"slack": "financialtimes/ft-origami"
 	},
-	"supportStatus": "active",
-	"ci": {},
+	"supportStatus": "experimental",
```
Comment (notlee) [inline origami.json]:
```suggestion
	"supportStatus": "active",
```
:shipit: !

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to b0d809db4eb167cf4288877b4ea5eb641bddf4aa
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

File: origami.json (hunk 5 above)
Comment (ker-an) [reply origami.json]: OMG I totally forgot - how exciting!!

**Comment by origamiserviceuser** [comment]
o-comments bundle size difference from 5.1.1 to 02934887bb613e1879c47b69b590d2e2db54fda5
js: 70.24kb decrease (-17.86kb/gzip)
css: 63.85kb decrease (-6.13kb/gzip)

**Pull request merged**